app/graphql/mutations/resources/create_resource.rb

```diff
@@ -0,0 +1,33 @@
+module Mutations
+    module Resources
+      class CreateResource < ::Mutations::BaseMutation
+        argument :name, String, required: true
+        argument :url, String, required: true
+        argument :folder_id, ID, required: true
+
+        def resolve(name:, url:, folder_id:)
+          img = get_image(url)
+          new_res = Resource.create(
+            name: name,
+            url: url,
+            image: img
+          )
+        
+          new_fr = FolderResource.create(
```

Comment:
could consider setting a folder variable and doing folder.resources.create(new_res.id), but that may be more coding style than anything else